packages/loader/container-loader/src/container.ts

```diff
@@ -748,7 +746,11 @@ export class Container
 		// Prefix all events in this file with container-loader
 		this.mc = loggerToMonitoringContext(ChildLogger.create(this.subLogger, "Container"));
 
-		this.options = cloneDeep(this.loader.services.options);
+		// Warning: this is only a shallow clone.  Mutation of any individual loader option will mutate it for
+		// all clients that were loaded from the same loader (including summarizer clients).
+		this.options = this.options = {
```

Comment:
```suggestion
		this.options = {
```

Reply:
Thanks, bad copy/paste

Reply:
Might want to enable https://eslint.org/docs/latest/rules/no-multi-assign